### PR TITLE
Add documentation to contributing for Graphviz, specifying the width …

### PR DESCRIPTION
**Pull request by stevepiercy**
…of media, and use cards to enhance the display of images.

Includes examples for other documentarians to copy-paste into their documentation. 💪

### CODE REVIEW

File: docs/contributing/myst-reference.md [hunk 1]
```diff
@@ -102,27 +102,54 @@ Use [Shimmer](http://example.com) for cleaner whiter teeth.
 #### Images and figures
```
Comment (ksuess) [inline docs/contributing/myst-reference.md]: Up to here it's cross references, from here on it's a new section, isn't it?
```suggestion
### Images and figures
```

File: docs/contributing/myst-reference.md [hunk 2]
```diff
@@ -102,27 +102,54 @@ Use [Shimmer](http://example.com) for cleaner whiter teeth.
...
 
 [Figures](https://docutils.sourceforge.io/docs/ref/rst/directives.html#figure) allow a caption and legend, whereas [images](https://docutils.sourceforge.io/docs/ref/rst/directives.html#images) do not.
+However we can {ref}`enhance images with cards <enhance-images-label>` to add a caption and more features.
 
 Use `image` for anything but diagrams.
 
 Use `figure` for diagrams.
 
 Paths to images and figures must resolve in both the main documentation and the submodule's documentation, if present.
```
Comment (ksuess) [inline docs/contributing/myst-reference.md]: What specifically does the contributor have to pay attention to?

**Review by ksuess — CHANGES_REQUESTED** [review]

File: docs/contributing/myst-reference.md (hunk 1 above)
Comment (stevepiercy) [reply docs/contributing/myst-reference.md]: Fixed this and several other instances in 4d27c5122ec9717cf78ccdb5dc155d356e047f7d.

File: docs/contributing/myst-reference.md (hunk 2 above)
Comment (stevepiercy) [reply docs/contributing/myst-reference.md]: I pushed two more commits to clean this up. Please take another look. Thank you!

**Review by ksuess — APPROVED** [review]
Awesome! Didn't know about Graphiz, that's cool.

**Comment by stevepiercy** [comment]
Graphviz came from @MrTango.

I've got a [fix in Volto](https://github.com/plone/volto/pull/4470) for the linkcheckbroken build here, so I'm going ahead to merge this one. Thank you for the review!

**Pull request merged**